deno/rest/v8/guild.ts

```diff
@@ -446,8 +448,22 @@ export interface RESTPatchAPICurrentGuildMemberNicknameJSONBody {
 	nick?: string | null;
 }
 
+/**
+ * https://discord.com/developers/docs/resources/guild#modify-current-member
+ */
+export interface RESTPatchAPICurrentGuildMemberJSONBody {
```

Comment:
This PR needs to be rebased to account for the changes in rest done to support TS 4.5!

Reply:
so what needs to be changed? just make it use the `AddUndefinedToPossiblyUndefinedPropertiesOfInterface`?

Reply:
Yep

Reply:
done